router/fastdp.go

```diff
@@ -267,9 +267,11 @@ func (fastdp *FastDatapath) bridge(ingress bridgePortID, key PacketKey, lock *fa
 		// If we did, we'd need to delete the flows every time
 		// we learned a new MAC address, or have a more
 		// complicated selective invalidation scheme.
+		log.Debug("bridge: unknown dst", ingress, key)
```

Comment:
We have two types of bridge. Maybe prefixing with "fastdp" would be less confusing?